src/components/Slider.react.js

```diff
@@ -18,14 +18,22 @@ export default class Slider extends Component {
     }
 
     render() {
-        const {id, loading_state, setProps, updatemode, vertical} = this.props;
+        const {
+            className,
+            id,
+            loading_state,
+            setProps,
+            updatemode,
+            vertical
+        } = this.props;
         const {value} = this.state;
         return (
             <div
                 id={id}
                 data-dash-is-loading={
                     (loading_state && loading_state.is_loading) || undefined
                 }
+                className={className}
```

Comment:
Apply class name to the wrapper. rc-slider applies very little styles to itself and relevant styles will be ok if applied to the parent (e.g. it doesn't explicitly set the color, font-size, etc.) unlike the dropdown.

Reply:
Most of the other cases where we allow `className`, we also allow `style` - do we want to do that here?

Reply:
No. This is because the wrapper was added with Loading Components and the wrapped slider did not support the style property. I don't mind adding support for styling here -- but will need to handle the existing  #481 style applied on the vertical slider.

Reply:
Fwiw, the rangeslider does not support `style` either.

Reply:
Given that the main purpose of `className` is styling, it seems appropriate to add `style`. But as that's a feature rather than a bugfix, feel free to make an issue and defer for later if you don't want to do it in this PR.